Ukhu/Banka
# #165448302 Implement LFA feedback on modal styling

#### What does this PR do?
Style's the different modals on the user interface based on feedback received from LFA

#### Description of Task to be completed?
- Style transaction details modal properly
- Style edit user modal properly
- Style credit/debit user modal properly

#### How should this be manually tested?
- Clone the repo
- Open the index.html file in the UI directory in the browser
- Navigate to the different pages with modals to ensure they are working properly

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#165448302

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
